src/Groups/perm.jl

```diff
@@ -220,6 +283,11 @@ function cperm(L::AbstractVector{T}...) where T <: IntegerUnion
    end
 end
 
+function cperm(L::Vector{Vector{T}}) where T <: IntegerUnion
```

Comment:
What's this about / this is nowhere documented?

Reply:
Sorry about that, I wrote documentation in the documentation for cperm(L::AbstractVector{T}...). I saw that that documentation was for both cperm(L::AbstractVector{<:T}...) where T <: IntegerUnion and cperm(G::PermGroup, L::AbstractVector{<:T}...). Should I also bring the cperm(G::PermGroup, L::AbstractVector{<:T}...) documentation to be directly above the function?